backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -70,6 +70,16 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.ExemptionRationale, src => src.GetNoteText(Entity.NoteTypes.Exemption))
                 .Map(dest => dest.ReportingNote, src => src.GetNoteText(Entity.NoteTypes.Reporting))
                 .Map(dest => dest.RemovalFromSplRationale, src => src.GetNoteText(Entity.NoteTypes.SplRemoval))
+                .Map(dest => dest.InterestFromEnhancedReferralNote, src => src.GetNoteText(Entity.NoteTypes.AgencyInterest))
+                .Map(dest => dest.DocumentationNote, src => src.GetNoteText(Entity.NoteTypes.Documentation))
```

Comment:
Flattening notes... I hate this approach, but I don't have time to modify the frontend to better handle arrays within Formik.